package.json

```diff
@@ -23,7 +23,8 @@
   "author": "felixge+thlorenz, adapted by Mike McNeil",
   "license": "MIT",
   "dependencies": {
-    "lodash": "3.10.1"
+    "lodash": "3.10.1",
```

Comment:
(or feel free to do it if you get to it before me- I just mean that there's no reason why that enhancement should hold up merging this or anything)